src/Playwright.Tests/NetworkPostDataTests.cs

```diff
@@ -52,7 +52,7 @@ public async Task ShouldReturnCorrectPostdataBufferForUtf8Body()
 
             await Task.WhenAll(task, actualTask);
 
-            string expectedJsonValue = JsonSerializer.Serialize(value, new()
+            string expectedJsonValue = JsonSerializer.Serialize(value, new JsonSerializerOptions
```

Comment:
the `new()` syntax fails because of a new overload to Serialize with 2 arguments.